src/datastore/postgres/PgDatastore.ts

```diff
@@ -265,18 +270,27 @@ export class PgDatastore implements Datastore, ClientEncryptionStore, Provisioni
         const userMessages: SchemaRunUserMessage[] = [];
         let currentVersion = await this.getSchemaVersion();
         while (currentVersion < PgDatastore.LATEST_SCHEMA) {
-            log.info(`Updating schema to v${currentVersion + 1}`);
+            let newVersion = currentVersion + 1;
+            if (currentVersion === 16 && PgDatastore.LATEST_SCHEMA === 161) {
+                newVersion = 161;
+            }
+            if (currentVersion === 161) {
+                newVersion = 17;
+            }
```

Comment:
We're using v161 instead of v17 so that we can incorporate a future v17 from upstream, and this code makes that happen. This is clearly hacky but gets the job done for now.